test/utilities.js

```diff
@@ -91,9 +91,10 @@ const registerTokens = async function(token_artifact, contract, token_owner, num
  */
 const setupEnvironment = async function(token_artifact, contract, token_owner, accounts, numTokens) {
   const tokens = await registerTokens(token_artifact, contract, token_owner, numTokens)
+  const amount = "100000000000000000000"
```

Comment:
nit: could use `web3.utils.toWei` as well?

Reply:
I tried using it here, but no web3 was available in this script... Didn't think it was worth the time to import it.  Do you?